x-pack/plugins/fleet/server/services/epm/elasticsearch/template/install.test.ts

```diff
@@ -193,11 +195,11 @@ describe('EPM index template install', () => {
 
     const packageTemplate = componentTemplates['metrics-package.dataset@package'].template;
 
-    if (!('mappings' in packageTemplate)) {
+    if (!('settings' in packageTemplate)) {
       throw new Error('no mappings on package template');
```

Comment:
```suggestion
      throw new Error('no settings on package template');
```

Reply:
also, it would be better to do an assert instead of throwing an error, something like
`expect(packageTemplate.settings).toBeDefined()`